### PR TITLE
working code... not working through insertion_sort

### PR DESCRIPTION
**Pull request by MISalz**

### CODE REVIEW

**Comment by MISalz** [comment]
- [ ] Top-level README “Table of Contents” is updated
 - [ x] README for this challenge is complete
       - [ ] Summary, Description, Approach & Efficiency, Solution
       - [ ] Picture of whiteboard
       - [ ] Link to code
 - [ ] Feature tasks for this challenge are completed
 - [ ] Unit tests written and passing
       - [ ] “Happy Path” - Expected outcome
       - [ ] Expected failure
       - [ ] Edge Case (if applicable/obvious)

File: python/code_challenges/insertion_sort.py [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+        num_arr[next_idx+1] = num_arr[next_idx]
+
+        next_idx -= 1
+        temp = num_arr[next_idx+1]
+        print(f"temp's current value: {temp}\n ****")
+
+  return num_arr
+
+insertion_sort(num_arr)
+
```
Comment (aaron-imbrock) [inline python/code_challenges/insertion_sort.py:26]: This code works which is great but I think you'd get more from following the pseudo code a bit closer.
Converted to python, it looks like this:
```
def insertionSort(arr):
	for i in range(1, len(arr)):
		j = i-1
		temp = arr[i]
		while (j >= 0 and temp < arr[j]):
			arr[j + 1] = arr[j]
			j = j - 1
		arr[j + 1] = temp
	return arr
```
